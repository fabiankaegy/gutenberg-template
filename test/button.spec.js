import {
	createNewPost,
	insertBlock,
	getEditedPostContent,
	visitAdminPage
} from "@wordpress/e2e-test-utils";

describe("Button can be inserted", () => {
	beforeEach(async () => {
		await visitAdminPage("/wp-admin");
		await createNewPost();
	});

	test("Can be added", async () => {
		await insertBlock("Button");
		expect(await getEditedPostContent()).toMatchSnapshot();
	});

	test("Can be added", async () => {
		await insertBlock("Example");
		expect(await getEditedPostContent()).toMatchSnapshot();
	});
});
