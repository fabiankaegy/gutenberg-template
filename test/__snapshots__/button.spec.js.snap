// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button can be inserted Can be added 1`] = `
"<!-- wp:button -->
<div class=\\"wp-block-button\\"><a class=\\"wp-block-button__link\\"></a></div>
<!-- /wp:button -->"
`;
